bamboo/common_python/tools.py

```diff
@@ -748,6 +753,17 @@ def test_func(cluster, executables, dir_name, compiler_name):
                                                'experiments',
                                                test_name)
 
+        # Setup the number of processes
+        if procs_per_node is not None:
+            assert 'procs_per_node' not in _kwargs.keys()
+            if procs_per_node == "auto":
+                procs_per_node = lbann.contrib.lc.systems.gpus_per_node()
```

Comment:
@timmoon10 should we change this to `lbann.contrib.systems.gpus_per_node()` so that any center gets it?

Reply:
I fixed it so that it recognizes other systems that are registered in the Python frontend.